### ISSUE TITLE
[Feature Request][Module Compatibility "Item Piles"] When an item piles is locked upon left click by the player make a message appear similar to that of the doors like "Item Piles is Locked"

### ISSUE THREAD

**Issue by p4535992** [Saibot393/LocknKey#21]
With my group of "newbies," we have finished testing the module and there are some things we could use (even on commission). I can also try to do a PR myself, but you tell me what you think first.

The use case: players get scared... clicking on the locked piles item nothing was happening (and that's correct), but they thought they had broke it and were going off on a wild goose chase with absurd checks...
Is it possible to add when the player left-clicks on an item piles locked a message clarifying that it is locked ?
In practice the same behavior as the doors where if the door is locked it writes "The door is Locked" and here instead "The item piles is locked"...

I see you already done the message for the lock picking and broken event.

**Comment by p4535992** [comment]
I ping @fantasycalendar (aka wasp) about it. Generally he puts hooks for everything. Maybe there is some hook you can use to check the opening event of the item piles.

**Comment by p4535992** [comment]
It should be this:

```
Hooks.on("item-piles-preLeftClickItem", (item, buttons, actor) => {
 // DO SOMETHING
});

Hooks.on("item-piles-preRightClickItem", (item, buttons, actor) => {
 // DO SOMETHING
});

```

**Comment by Saibot393** [comment]
I just released v1.9.4 which should enable the "popups on interaction" for item piles
Thanks for the code above, eventhough i did not use it in the end, it helped me improve the item piles integration which in the end allowed me to enabled the popups (i previously used the "enabled" setting to lock item piles, the problem here was, that not enabled item piles no longer call hooks, now locked item piles stay enabled but will no longer open their item sheet (unless a GM opens them))
